packages/ncids-css/scss/components/nci-cta-strip/internal/_cta-buttons.scss

```diff
@@ -0,0 +1,17 @@
+.nci-cta-button {
```

Comment:
This should be an element, e.g. `.nci-cta-strip__button` if it is not just going to be a `usa-button` for the element.

Reply:
also if you are using the "primary" style as indicated with the theme token, you should be importing that style definition.